src/test/regress/expected/multi_create_shards_2.out

```diff
@@ -0,0 +1,238 @@
+SET citus.next_shard_id TO 370000;
```

Comment:
Again, the diff is only with OIDS,
```
43a44,46
> ERROR:  syntax error at or near "WITH"
> LINE 1: ALTER TABLE table_to_distribute SET WITH OIDS;
>                                             ^
45,46c48,49
< ERROR:  cannot distribute relation: table_to_distribute
< DETAIL:  Distributed relations must not specify the WITH (OIDS) option in their definitions.
---
> ERROR:  cannot create constraint on "table_to_distribute"
> DETAIL:  Distributed relations cannot have UNIQUE, EXCLUDE, or PRIMARY KEY constraints that do not include the partition column (with an equality operator if EXCLUDE).

```

Just removing the test that errors out is much better compared to adding a new file. The test already does a trivial check, so we can live without that test.